src/components/Popup/Popup.tsx

```diff
@@ -0,0 +1,27 @@
+import Popup from 'reactjs-popup';
+import 'reactjs-popup/dist/index.css';
+import Style from './Popup.module.scss';
+import Image from 'next/image';
+
+export default function Modal({
+  isOpen,
+  errorText,
+}: {
+  isOpen: boolean;
+  errorText: string;
```

Comment:
Please rename `errorText` to `text` and add another `status` prop. Then you can use `status` `error` to change styles accordingly and you’ll be able to reuse this modal for more cases rather than just for errors